c2rust-transpile/src/c_ast/mod.rs

```diff
@@ -973,8 +973,10 @@ impl TypedAstContext {
         }
     }
 
-    pub fn sort_top_decls(&mut self) {
-        // Group and sort declarations by file and by position
+    /// Sort the top-level declarations by file and source location
+    /// so that we preserve the ordering of all declarations in each file.
+    /// This preserves the order when we emit the converted declarations.
+    pub fn sort_top_decls_for_emitting(&mut self) {
```

Comment:
Nitpick: I'm not sure this function name needed to change.

Reply:
It's to contrast with `fn sort_top_decls_for_converting` in #1372.